### PR TITLE
SONAR-7532 do not log submitter when unset

### PR DESCRIPTION
**Pull request by sns-seb**

### CODE REVIEW

File: server/sonar-server/src/main/java/org/sonar/server/computation/taskprocessor/CeWorkerCallableImpl.java [hunk 1]
```diff
@@ -97,24 +97,26 @@ private void executeTask(CeTask task) {
 
   private static Profiler startProfiler(CeTask task) {
     Profiler profiler = Profiler.create(LOG);
-    return profiler.startInfo("Execute task | project={} | type={} | id={} | submitter='{}'",
-      task.getComponentKey(), task.getType(), task.getUuid(), getSubmitterLoginForLog(task));
+    return profiler.startInfo("Execute task | project={} | type={} | id={}{}",
+      task.getComponentKey(), task.getType(), task.getUuid(), submitterSuffix(task));
```
Comment (simonbrandhof) [inline server/sonar-server/src/main/java/org/sonar/server/computation/taskprocessor/CeWorkerCallableImpl.java]: You can simply use `profiler.addContext(...)` !

Comment (sns-seb) [inline server/sonar-server/src/main/java/org/sonar/server/computation/taskprocessor/CeWorkerCallableImpl.java]: indeed, it looks better

Comment (sns-seb) [inline server/sonar-server/src/main/java/org/sonar/server/computation/taskprocessor/CeWorkerCallableImpl.java]: hum... but time is now the first "parameter", not the last one...

Comment (simonbrandhof) [inline server/sonar-server/src/main/java/org/sonar/server/computation/taskprocessor/CeWorkerCallableImpl.java]: is it so important ? I don't think.

**Comment by sns-seb** [comment]
@simonbrandhof improved `Profiler` to log time after context and so, used it in `CeWorkerCallableImpl`

**Comment by simonbrandhof** [comment]
LGTM

File: server/sonar-server/src/main/java/org/sonar/server/computation/taskprocessor/CeWorkerCallableImpl.java (hunk 1 above)
Comment (sns-seb) [inline server/sonar-server/src/main/java/org/sonar/server/computation/taskprocessor/CeWorkerCallableImpl.java]: yes, for parsing, time is the most relevant information, having it at the end make parsing easier

**Pull request merged**